pkg/plugin/twinmaker/utils.go

```diff
@@ -353,7 +353,6 @@ func (s *twinMakerHandler) GetComponentHistoryWithLookupHelper(ctx context.Conte
 								}
 							}
 						}
-						break
```

Comment:
why was this removed?

Reply:
We have multiple alarm component types now.  This break causes a premature return after one is matched leaving the others processed.